application/config/config.sample.php

```diff
@@ -74,6 +74,7 @@
 $config['auth_mode'] = '3';
 
 $config['auth_level'][3] = 'Operator';
+$config['auth_level'][5] = 'Special Callsign';
```

Comment:
Please add this line in `install/config/config.php` aswell

Reply:
`5` was more or less random. But will change that to `2`.
I like the idea with the comment, need to figure out how to edit the wiki once this is merged / released.

Reply:
The Wiki is open to edit for everyone

https://github.com/wavelog/wavelog/wiki/Recommended-Setup-for-Special-Callsigns-and-Clubs

Reply:
Ah that doesn't work. That would disable api access which would suck when you use wsjt-x.

Reply:
@derFogel I don't see an issue to change API access to `authorize(2)` but the others have to agree with that

Reply:
Actually the api (I tested with the radio api) works when I change it to 2

Reply:
With a permission level of `2` you cannot log anything, so this will be a bigger change.

Reply:
my 2ct on that:
i like the idea to configure it as well.
but imho an option (via the user_options_model) may fit better here.

reasons:
- The grants (in a classical sense) are still the same for those stations.
- it's "only" the popup which is shown to those special-logins. logic is completly the same like a normal user has (as a "normal user" you can set the OP-Call at the first QSO - see station_tab. OP is persitant for the session
- with the user_option we don't touch the config.php - it is minimal-invasive compared to a new Userlevel